.gitignore

```diff
@@ -36,6 +36,7 @@ logs/*
 npTicket*
 r.tar.gz
 lighthouse.yml.configme
+chatCensoredList.txt
```

Comment:
```suggestion
```
If we aren't doing the git method anymore the chatCensoredList doesn't need to be in the gitignore and you could also remove the file from the project altogether since it would be handled by the code.